audinterface/core/feature.py

```diff
@@ -278,12 +305,13 @@ def add_unit(dur, unit):
         feature_names = audeer.to_list(feature_names)
         if num_channels > 1:
             column_names = []
-            for channel in range(num_channels):
```

Comment:
So far we had always named the channels `0, 1, ...`, but I think it makes more sense to stick to the actual channel IDs. That way you will later see directly from the column names which channels of the file were processed.